stats/shot_chart_detail.py

```diff
@@ -0,0 +1,144 @@
+import urllib.parse
+
+from models import ShotChartDetail, ShotChartDetailTemp, Game
+from general_requester import GenericRequester
+from peewee import JOIN
+
+
+class ShotChartDetailRequester(GenericRequester):
+
+    shot_char_detail_url = "https://stats.nba.com/stats/shotchartdetail"
+
+    def __init__(self, settings):
+        """
+        Constructor. Pass on all relevant vars.
+        """
+        super().__init__(settings, self.shot_char_detail_url, ShotChartDetail)
+
+    def create_ddl(self):
+        """
+        Override method to setup temp table.
+        """
+        super().create_ddl()
+        self.settings.db.bind([ShotChartDetailTemp])
+        self.settings.db.create_tables([ShotChartDetailTemp], safe=True)
+
+    def finalize(self):
+        """
+        This function finishes loading shot_chart_detail by inserting all valid
+        records from the temp table into the main table, then dropping the temp
```

Comment:
Where is the call to drop the temp table?

Reply:
Good eye, but this uses a table specified with `TEMPORARY` will be dropped at the end of the session. That is defined in the `Meta` section of the Model.

Reply:
I'll fix the comment.

Reply:
Ahh, that's very cool.